index.html

```diff
@@ -0,0 +1,27 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>Document</title>
+    <link rel="stylesheet" href="style.css" />
+  </head>
+  <body>
+    <h1>Peer Review</h1>
+    <p>
+      This is just a dummy site to review each other's work. I used limited
+      styling and simple HTML code to create this website for the sole purpose
+      of review. <br />
+      I see this review execise to be very informative and illuminating because
+      it forces you to write clean codes and adopt professional coding habits
+      and practises.
+    </p>
+
+    <p>
+      Lorem ipsum dolor sit amet, consectetur adipisicing elit. Minus quisquam,
+      distinctio sapiente voluptas labore asperiores. Consectetur sed deserunt
+      enim laudantium voluptates dolores facilis in alias omnis, iste aut culpa
```

Comment:
You have done very well on this HTML document, but your code's randomly placed blank spaces make reading a bit troublesome. I suggest you keep it consistent with where you place your blank lines. For instance, you can space out sibling elements with blank lines to make your code more readable instead of placing them right after opening tags.